website/content/en/preview/getting-started/getting-started-with-karpenter/scripts/step02-create-cluster.sh

```diff
@@ -24,7 +24,7 @@ iam:
       namespace: karpenter
     roleName: ${CLUSTER_NAME}-karpenter
     attachPolicyARNs:
-    - arn:aws:iam::${AWS_ACCOUNT_ID}:policy/KarpenterControllerPolicy-${CLUSTER_NAME}
+    - arn:${AWS_PARTITION}:iam::${AWS_ACCOUNT_ID}:policy/KarpenterControllerPolicy-${CLUSTER_NAME}
```

Comment:
Looks like there are two more occurrences on line 31 and line 53, can you add those in?